DS/sql1create.sql

```diff
@@ -0,0 +1,243 @@
+INSERT INTO publisher(publisher_name)VALUES('NEGI');
+INSERT INTO publisher(publisher_name)VALUES('SHARMA');
+INSERT INTO publisher(publisher_name)VALUES('SHARMA-PATEL');
+
+SELECT * FROM publisher;
+
+INSERT INTO subjects(subject_id,subject_name)VALUES(111,'SOCAIL');
+INSERT INTO subjects(subject_name)VALUES('BIO');
+INSERT INTO subjects(subject_name)VALUES('Physics');
+INSERT INTO subjects(subject_name)VALUES('Soil');
+
+SELECT * FROM subjects;
+
+INSERT INTO authors(author_id,author_name)VALUES(10001,'A.K.ROV');
+INSERT INTO authors(author_name)VALUES('K.K.ROV');
+INSERT INTO authors(author_name)VALUES('L.K.MISHRA');
+INSERT INTO authors(author_name)VALUES('ARUN SHARMA');
+
+SELECT * FROM authors;
+
+INSERT INTO titles(title_id,title_name,subject_id,publisher_id)VALUES(11,'Horror',111,1001);
+INSERT INTO titles(title_name,subject_id,publisher_id)VALUES('Horror',112,1002);
+INSERT INTO titles(title_name,subject_id,publisher_id)VALUES('Social',113,1001);
+INSERT INTO titles(title_name,subject_id,publisher_id)VALUES('Social',114,1004);
+
+SELECT * FROM titles;
+
+INSERT INTO books(accession_no,title_id,purchase_date,price,status)VALUES(101,12,'2004-06-04',1000,'AVAILABEL');
+INSERT INTO books(title_id,purchase_date,price,status)VALUES(12,'2009-06-17',1000,'NOT AVAILABEL');
+INSERT INTO books(title_id,purchase_date,price,status)VALUES(14,'2002-07-04',1000,'AVAILABEL');
+INSERT INTO books(title_id,purchase_date,price,status)VALUES(11,'2007-01-24',1000,'NOT AVAILABEL');
+
+SELECT * FROM books;
+
+INSERT INTO book_issue(accession_no,member_id)VALUES(101,2);
+INSERT INTO book_issue(accession_no,member_id)VALUES(103,3);
+INSERT INTO book_issue(accession_no,member_id)VALUES(102,4);
+INSERT INTO book_issue(accession_no,member_id)VALUES(102,1);
+
+
+SELECT * FROM book_issue;
+
+INSERT INTO book_return(return_date,accession_no,member_id,issue_date)VALUES('2015-09-23',101,2,'2015-09-08');
+INSERT INTO book_return(return_date,accession_no,member_id,issue_date)VALUES('2015-09-23',103,1,'2015-09-08');
+INSERT INTO book_return(return_date,accession_no,member_id,issue_date)VALUES('2015-09-23',102,4,'2015-09-08');
+
+SELECT * FROM book_return;
+
+INSERT INTO title_author(title_id,author_id)VALUES(11,10001);
+INSERT INTO title_author(title_id,author_id)VALUES(12,10002);
+INSERT INTO title_author(title_id,author_id)VALUES(13,10002);
+INSERT INTO title_author(title_id,author_id)VALUES(14,10004);
+
+SELECT * FROM title_author;
+
+UPDATE members SET addressline2="JAIPUR";
+
+SELECT * FROM members;
+
+UPDATE members SET addressline1="EPIP Sitapura" where category='F';
+
+SELECT * FROM members;
+
+DELETE FROM publisher;
+
+SELECT * FROM publisher;
+
+SET @publication1:='SHARMA';
+SET @publication2:='JAIN';
+SET @publication3:='MITTAL';
+SET @publication4:='PANDIT'; 
+
+SET @pblisherID1:=1001;
+SET @pblisherID2:=1002;
+SET @pblisherID3:=1003;
+SET @pblisherID4:=1004;
+
+INSERT INTO publisher(publisher_id,publisher_name)VALUES(@pblisherID1,@pblisherID1);
```

Comment:
ACTION : You are assigning pblisherID1  to both publisher_id and publisher_name then what is the point of creating publication1 when you are not using it.You should assign ID to id field and publication1 to name field.